src/main/java/seedu/address/model/person/Person.java

```diff
@@ -113,6 +135,7 @@ public String toString() {
                 .add("email", email)
                 .add("address", address)
                 .add("tags", tags)
+                .add("isPinned", String.valueOf(isPinned))
```

Comment:
With the above suggestion, this could be replaced with the PinStatus' toString method which could potentially return "pinned", "unpinned", or any other possible pin statuses which could be implemented in the future.